src/fut/mod.rs

```diff
@@ -52,35 +52,77 @@ use crate::actor::Actor;
 /// then updating the initiating actor's state:
 ///
 /// ```rust,ignore
-/// impl Message for SomeMessage {
-///     type Result = ();
+/// // The response type returned by the actor future
+/// type OriginalActorResponse = ();
+/// // The error type returned by the actor future
+/// type MessageError = ();
+/// // This is the needed result for the DeferredWork message
+/// // It's a result that combine both Response and Error from the future response.
+/// type DeferredWorkResult = Result<OriginalActorResponse, MessageError>;
+/// 
+/// # struct ActorState {}
+/// 
+/// # impl ActorState {
+/// #    fn update_from(&mut self, _result: ()) {}
+/// # }
+/// 
+/// # struct OtherActor {}
+/// 
+/// # impl Actor for OtherActor {
+/// #    type Context = Context<Self>;
+/// # }
+/// 
+/// # impl Handler<OtherMessage> for OtherActor {
+/// #    type Result = ();
+/// #
+/// #    fn handle(&mut self, _msg: OtherMessage, _ctx: &mut Context<Self>) -> Self::Result {
+/// #    }
+/// # }
+/// 
+/// # struct OriginalActor{
+/// #     other_actor: Addr<OtherActor>,
+/// #     inner_state: ActorState
+/// # }
+/// 
+/// # impl Actor for OriginalActor{
+/// #     type Context = Context<Self>;
+/// # }
+/// 
+/// # struct DeferredWork{}
+/// 
+/// impl Message for DeferredWork {
+///     type Result = DeferredWorkResult;
 /// }
-///
+/// 
+/// # #[derive(Message)]
+/// # struct OtherMessage{}
+/// 
 /// impl Handler<DeferredWork> for OriginalActor {
 ///     // Notice the `Response` is an `ActorFuture`-ized version of `Self::Message::Result`.
-///     type Result = ResponseActFuture<Self, (), Error>;
-///
-///     fn handle(&mut self, _msg: Refresh, ctx: &mut Context<Self>) -> Self::Result {
+///     type Result = ResponseActFuture<Self, OriginalActorResponse, MessageError>;
+/// 
```

Comment:
There is a trailing space, should be removed.
```suggestion
///
```